routing/router_test.go

```diff
@@ -521,7 +548,7 @@ func TestSendPaymentErrorRepeatedFeeInsufficient(t *testing.T) {
 	}
 
 	// Craft a LightningPayment struct that'll send a payment from roasbeef
-	// to luo ji for 100 satoshis.
+	// to sophon for 1000 satoshis.
```

Comment:
I think the fix to this existing test can be moved to a separate commit, stacking on top of the priv key map commit.

Reply:
I prefer to leave it as is. Its a pain to extract those changes, and I think the fix is aligned with the signature checks that were added to the new test, so its not an unrelated. Anyhow, if you believe it's important let me know and I'll extract to a separate commit.

Reply:
If you remove the mission control policy update code and `TestSendPaymentErrorFeeInsufficientPrivateEdge` (and leaving `signErrChanUpdate` in), I think a clean commit remains that fixes the existing test. And then commit on top of that the new functionality and its tests.

In know spending time on clean commit history can feel like a pain. But a large part of it is getting used to it, using the right tools and understanding git rebase commands. I use vscode for this sometimes. With all changes in the working directory, it allows you to select chunks of files and stage only those. There are other editors that offer similar functionality.

The basic assumption here is that it indeed takes the author of the PR extra time (but with some experience this can be marginal) than just creating a single big commit, but more time is saved by the (at least two) people who review the code and all future developers looking up this commit to understand the greater picture of a particular change.

Reply:
Ok. Done

Reply:
Thanks. For the non-author it really makes it much easier to understand what parts of the full set of changes relate to each other.